manuscript/glossary.md

```diff
@@ -0,0 +1,80 @@
+### Glossary and Abbreviations used in the manuscript
+
+#### Projects
+
+* **GBIF** (Global Biodiversity Information Facility, http://www.gbif.org) provides
+  an international open data infrastructure to publish and disseminate
+  biodiversity information.
+
+* **iDigBio** (Integrated Digitized Biocollections, https://www.idigbio.org) is a
+  project funded by the National Science Foundation that facilitates the
+  digitization of natural history collections, and provides data and images for
+  biological specimens.
+
+#### File Formats
+
+* **CSV** (Comma-Separated Values) and **TSV** (Tab-Separated Values) are plain text
+  file formats used to store tabular data where each row is represented by a line
+  in the file, and each field (column) is separated by a comma for CSV or by the
+  Tab character for TSV.
+
+* **FASTA** is a simple file format used to represent sequences of nucleotides or
+  amino-acids in plain text making it easy to manipulate programmatically.
+
+* **HDF5** (Hierarchical Data Format) is an open-source binary file format designed
+  to store large amount of data (and their associated metadata) by providing a
+  hierarchical structure that could be compared to how a hard drive is organized
+  with directories and files.
+
+* **NetCDF** (Network Common Data Form) is an open-source binary file format
+  designed to store large datasets in array-oriented scientific data as
+  typically used in geosciences.
+
```

Comment:
yeah, so, since TSV is no less esoteric than FASTA, HDF5, etc, TSV seems in scope ...